### PR TITLE
Done, but I am not sure if everything is correct, OOP is really compl…

### PR DESCRIPTION
**Pull request by vovkandrew**
…icated to understand for me

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,19 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    Machine trackOne = new Track();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/MainApp.java:15]: you should create main method here and do all of this there

File: src/main/java/core/basesyntax/Track.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Track extends Machine {
+    public void setTrackOne(Machine trackOne) {
+    }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Track.java:5]: Could you explain what does this method?

File: src/main/java/core/basesyntax/MainApp.java (hunk 1 above)
Comment (VovkAA) [reply src/main/java/core/basesyntax/MainApp.java]: Fixed, now should be correct

File: src/main/java/core/basesyntax/Track.java (hunk 2 above)
Comment (VovkAA) [reply src/main/java/core/basesyntax/Track.java]: Still trying to understand oop, apparently, I misunderstood constructor and setter

**Pull request closed without merging**